lectures/cagan_adaptive.md

```diff
@@ -16,13 +16,13 @@ kernelspec:
 ## Introduction
 
 
-This lecture is a sequel or prequel to another lecture {doc}`monetarist theory of  price levels <cagan_ree>`.
+This lecture is a sequel or prequel to another lecture {doc}`A monetarist theory of  price levels <cagan_ree>`.
```

Comment:
Thanks @longye-tian .

How about "This lecture is a sequel or prequel to the lecture {doc}`cagan_ree`.